test/integration/test.cancelContentPlay.js

```diff
@@ -110,8 +110,8 @@ QUnit.test("cancelContentPlay doesn\'t block play in content playback", function
   this.player.trigger('play');
   assert.strictEqual(pauseSpy.callCount, 1, 'pause should have been called');
   assert.strictEqual(
-    this.player.ads._cancelledPlay, true,
-    'cancelContentPlay is called while resuming'
```

Comment:
So, simply changing this test suggests that the behavior of the library changed. Instead of changing the test, we should investigate why this behavior changed.

Reply:
Following our Slack discussion, we may want to try firing both `play` and `playing` 🤔